README.md

```diff
@@ -16,6 +16,7 @@ This repository contains samples for creating Kubernetes Applications in Azure M
 | Sample | Description |
 |--------|-------------|
 |[k8s-offer-azure-vote](samples/k8s-offer-azure-vote/) | The Azure Vote application is a simple sample used throughout the Kubernetes Applications Marketplace documentation. The application consists of two pods, one running a flask web form, and the second a redis instance for temporary data storage. |
+|[k8s-existing-cluster-only-sample](samples/k8s-existing-cluster-only-sample/) | This sample shows how ISV can use only existing cluster option in their create ui definition. |
```

Comment:
>This sample shows how ISV can use only existing cluster option in their create ui definition. [](http://example.com/codeflow?start=81&length=93)

nit: Can you please change this text to below?

This sample illustrates how ISV can allow an application to deploy only on existing clusters.